apps/nextra/pages/en/build/cli/use-cli/working-with-move-contracts/local-simulation-benchmarking-and-gas-profiling.mdx

```diff
@@ -0,0 +1,251 @@
+        create_ty                                                 0.0004      0.01%
+        create_ty                                                 0.0004      0.01%
+        create_ty                                                 0.0004      0.01%
+        create_ty                                                 0.0004      0.01%
+        create_ty                                                 0.0008      0.02%
+        imm_borrow_loc                                            0.00022     0.01%
+        call                                                      0.00441     0.14%
+        0x1::signer::address_of                                   0.007534    0.23%
+            create_ty                                             0.0008      0.02%
+            move_loc                                              0.000441    0.01%
+            call                                                  0.004043    0.12%
+            0x1::signer::borrow_address                           0.000735    0.02%
+            read_ref                                              0.001295    0.04%
+            ret                                                   0.00022     0.01%
+        st_loc                                                    0.000441    0.01%
+        copy_loc                                                  0.000854    0.03%
+        load<0xdbcb..::0xdbcb..::message::MessageHolder>          0.302385    9.33%
+        exists_generic                                            0.000919    0.03%
+        not                                                       0.000588    0.02%
+        br_false                                                  0.000441    0.01%
+        imm_borrow_loc                                            0.00022     0.01%
+        move_loc                                                  0.000441    0.01%
+        pack                                                      0.000955    0.03%
+        move_to_generic                                           0.001838    0.06%
+        branch                                                    0.000294    0.01%
+        @28                                                       
+        ret                                                       0.00022     0.01%
+    ledger writes                                                 0.097756    3.01%
+        transaction                                               
+        events                                                    
+        state write ops                                           0.097756    3.01%
+            create<0xdbcb..::0xdbcb..::message::MessageHolder>    0.097756    3.01%
+```
+
+The left column lists all Move instructions and operations being executed, with each level of indentation indicating a function call.
+
+The middle column represents the gas costs associated with the operations.
+
+There is also a special notation `@number` that represents a jump to a particular location in the byte code.
```

Comment:
curious, are there also line numbers in this table? to understand where `@number` jumps to?

Reply:
The number here is the offset to a bytecode location, so it can't be easily shown here. In the future we should just go a step further and show people annotated source code.